community/CloudflareR2.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: page
+title: Hosting BlueMap on Cloudflare R2
+parent: Community Guides
+nav_order: 15
+---
+
+# How to host BlueMap on Cloudflare R2
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+## What is Cloudflare R2?
+
+Cloudflare R2 is a service that allows you to host files on Cloudflare's network. With a couple of tweaks, you can host static websites such as BlueMap on Cloudflare R2. Please bear in mind that Cloudflare R2 costs money.
+
+## What do you need for this?
+
+- Cloudflare account
+- Domain you own & is connected with Cloudflare
+- Credit card/prepaid card you have
+- a completely rendered BlueMap map (you can use the CLI or the docker container for that) — this is intended as a one-time upload for a map where further adjustments are unnecessary
```

Comment:
Maybe it'd be good to highlight and emphasise this requirement in the first paragraph, to really ensure people who want live maps don't try to use this.

Reply:
Changed - added two sentences at the top of who this guide is for✔